Source/Core/Core/HW/WiimoteReal/IOWin.cpp

```diff
@@ -395,15 +502,18 @@ void WiimoteScanner::CheckDeviceType(std::basic_string<TCHAR> &devicepath, bool
 		u8 const disable_enc_pt2_report[MAX_PAYLOAD] = {WM_SET_REPORT | WM_BT_OUTPUT, WM_WRITE_DATA, 0x04, 0xa4, 0x00, 0xfb, 0x01, 0x00};
 
 		CheckDeviceType_Write(dev_handle,
+							  write_method,
```

Comment:
The other arguments are indented by two tabs and a lot of spaces, whereas my addition follows the coding style with multiple tabs and then spaces for the alignment.